File: fmarket/src/main/webapp/app/models/demandAdminDto.ts
import {DemandStatus} from "./DemandStatus";
/**
 * Created by NicolaeB on 4/26/2016.
 */
export class DemandAdminDto{
    public id:number;

    public title:string;

    public accountId:string;

    public creationDate:Date;

    public status:DemandStatus;//to do find a way to map this to a enum string
}

File: fmarket/src/main/java/ro/fmarket/admin/demand/DemandAdminDTO.java
package ro.fmarket.admin.demand;

import org.joda.time.DateTime;

import lombok.Data;
import ro.fmarket.model.demand.consts.DemandStatus;

@Data
public class DemandAdminDTO {

	private int id;
	
	private String title;
	
	private int accountId;
	
	private DateTime creationDate;
	
	private DemandStatus status;
	
}
